src/labels/CustomLabels.labels

```diff
@@ -12731,4 +12731,107 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
+        <shortDescription>gsLearnMore</shortDescription>
+        <value>Learn More</value>
+    </labels>
+
+    <labels>
+        <fullName>gsDaysRemainingInFreeTrial</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsDaysRemainingInFreeTrial</shortDescription>
+        <value>Days Remaining in Free Trial</value>
+    </labels>
+
+    <labels>
+        <fullName>gsApplyForFreeLicenses</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsApplyForFreeLicenses</shortDescription>
+        <value>Apply for Your Salesforce Licenses</value>
+    </labels>
+
+    <labels>
+        <fullName>gsApplicationStatus</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsApplicationStatus</shortDescription>
+        <value>Application Status: </value>
+    </labels>
+
+    <labels>
+        <fullName>gsSubmitted</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsSubmitted</shortDescription>
+        <value>Submitted</value>
+    </labels>
+
+    <labels>
+        <fullName>gsDaysAdded</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsDaysAdded</shortDescription>
+        <value>30 Days Added</value>
+    </labels>
+
+    <labels>
+        <fullName>gsCheckStatus</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsCheckStatus</shortDescription>
+        <value>Check Status</value>
+    </labels>
+
+    <labels>
+        <fullName>gsApplicationStatusModalHeader</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsApplicationStatusModalHeader</shortDescription>
+        <value>Check Application Status</value>
+    </labels>
+
+    <labels>
+        <fullName>gsClose</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsClose</shortDescription>
+        <value>Close</value>
+    </labels>
+
+    <labels>
+        <fullName>gsFollowUpApplicationStatus</fullName>
+        <categories>gettingStarted</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>gsFollowUpApplicationStatus</shortDescription>
+        <value>To follow up on your application status, please email us at: something@salesforce.com</value>
```

Comment:
We need a real email address here. According to the website, it should be powerofus@salesforce.com.